src/autogain/gstpeautogain.cpp

```diff
@@ -439,12 +467,23 @@ static void gst_peautogain_process(GstPeautogain* peautogain, GstBuffer* buffer)
 
     if (!failed) {
       if (peautogain->use_geometric_mean) {
-        peautogain->loudness = std::cbrt(peautogain->momentary * peautogain->shortterm * peautogain->global);
+        if (peautogain->use_static_integrated_value) {
+          peautogain->loudness = std::cbrt(peautogain->momentary * peautogain->shortterm * peautogain->staticv);
+        } else {
+          peautogain->loudness = std::cbrt(peautogain->momentary * peautogain->shortterm * peautogain->global);
+        }
       } else {
-        peautogain->loudness =
-            (peautogain->weight_m * peautogain->momentary + peautogain->weight_s * peautogain->shortterm +
-             peautogain->weight_i * peautogain->global) /
-            (peautogain->weight_m + peautogain->weight_s + peautogain->weight_i);
+        if (peautogain->use_static_integrated_value) {
+          peautogain->loudness =
+              (peautogain->weight_m * peautogain->momentary + peautogain->weight_s * peautogain->shortterm +
```

Comment:
maybe refactor? We can just replace integrated with static value by passing it to parameter of function, so we don't need 2x nearly same code.